QRScanner/QRScannerView.swift

```diff
@@ -49,6 +55,9 @@ public class QRScannerView: UIView {
     @IBInspectable
     public var animationDuration: Double = 0.5
 
+    @IBInspectable
+    public var blurEffectEnable = false
```

Comment:
👍

Reply:
I think `isBlurEffectEnabled` is better name.
This is boolean.

Reply:
Renamed in https://github.com/mercari/QRScanner/pull/13/commits/45c3c4515788a13737a701fcc336d60a75a300b0